README.md

```diff
@@ -72,7 +72,129 @@ const wertWidget = new WertWidget(options);
 
 ### Listeners
 
-_Coming soon..._
+There is an ability to listen some widget events in order to react on them. Whole list of available events one can get by calling static property **eventTypes** on helper:
+
+```
+import WertWidget from '@wert-io/widget-initializer';
+
+console.log(WertWidget.eventTypes);
+
+/* console output:
+[
+  'close',
+  'error',
+  'loaded',
+  'payment-status',
+  'position',
+]
+*/
+```
+
+Subscribing:
+
+```
+const widget = new WertWidget({
+  ...options,
+  listeners: {
+    position: data => console.log('step:', data.step),
+  },
+});
+```
+
+Expected data by event:
+
+<table>
+  <tr>
+    <th>Event type</th>
+        <th>Data</th>
+        <th>Description</th>
+  </tr>
+    <tr>
+<td>
+
+`close`
+</td>
+<td>
+
+```
+undefined
+```
+</td>
+<td>
+Event threw by widget when it's time to close it (by flow logic). Widget won't be closed by itself — it only throws event.
+</td>
+    </tr>
+    <tr>
+<td>
+
+`error`
+</td>
+<td>
+
+```
+{
+  name: String
+    message: String
+}
+```
+</td>
+<td>
+Information about error occurred in widget.
+</td>
+    </tr>
+    <tr>
+<td>
+
+`loaded`
+</td>
+<td>
+
+```
+undefined
+```
+</td>
+<td>
+Widget got necessary data and either ready to work or to receive extra data (if it was used). Can be duplicated because of 3DS redirects.
```

Comment:
The module has the necessary data and is ready to work. In case extra data was used, it's ready to receive it. This event can be duplicated if there was a 3D Secure redirection.